packages/dom-test-kit/src/stylable-dom-util.ts

```diff
@@ -14,8 +14,16 @@ export interface PartialElement {
     classList: Element['classList'];
 }
 
+export type StylesheetHost = {
```

Comment:
is this used any where now?

Reply:
I think not, should I fix (and use) or remove this?

Reply:
Choose to fix.